packages/experiments/src/components/Chiclet/ChicletXsmall.noImage.test.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
```

Comment:
We can combine this no image test and the image test into one test file! Here's a good example of how Spinner does it: [Spinner.test.tsx](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Spinner/Spinner.test.tsx).